identityClient/identityClient.go

```diff
@@ -676,6 +676,18 @@ func (c *E3dbIdentityClient) RealmInfo(ctx context.Context, realmName string) (*
 	return info, err
 }
 
+// PrivateRealmInfo fetches the private realm information based on realm name.
+func (c *E3dbIdentityClient) PrivateRealmInfo(ctx context.Context, realmName string) (*PrivateRealmInfo, error) {
+	var info *PrivateRealmInfo
+	path := c.Host + identityServiceBasePath + "/realm/info" + strings.ToLower(realmName)
```

Comment:
The endpoint should handle lower casing the realm name, having `strings.ToLower` is an anti pattern / code smell

Reply:
(updated)